articles/storage/storage-use-azcopy.md

```diff
@@ -250,6 +250,15 @@ After the copy operation, the target container will include the blob and its sna
     abc (2013-02-25 080757).txt
     abc (2014-02-21 150331).txt
 
+### Copy all blobs across Storage accounts
```

Comment:
This title implies that this copies all the blobs in one storage account to another storage account, and that's not what it does. Can you please change this to "Copy all blobs in a container to another Storage account"?